CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+Hi! We are really excited that you are interested in contributing to Gatsby and its related projects part of the [Gatsby organization](https://github.com/gatsbyjs).
+
+Please take a moment to review this document in order to make the contribution process easy and effective for everyone involved.
+
+## Code of Conduct
+
+Help us keep Gatsby open and inclusive—please read and follow our [Code of Conduct](https://www.gatsbyjs.org/code-of-conduct/).
+
+## Issues and general contributing guidelines
+
+Please take a look at our [general contributing guidelines](https://www.gatsbyjs.org/docs/how-to-contribute/) for the main Gatsby repository for universal information on [filing an issue](https://www.gatsbyjs.org/docs/how-to-contribute/#filing-an-issue) and [contributing](https://www.gatsbyjs.org/docs/how-to-contribute/#contributing).
+
+## Pull requests
+
+Good pull requests—bug fixes, improvements, new features—are a great help. Gatsby would not be what it is without the contributions of our community. Please consider the following guidelines:
+
+- Please take a moment before starting to work on any significant pull request (e.g. implementing features or refactoring code) and find out whether your idea fits with the scope and aims of the project. If you are unsure, please consider opening a new issue to discuss your idea before you do any significant work that the project's developers might not want to merge into the project.
+- Pull requests should remain focused in scope and avoid containing unrelated commits.
+- It is fine to have multiple, smaller commits as you work on the PR. We will let GitHub automatically squash your individual commits before merging.
+- *If you are adding new feature*:
+  - Please provide a convincing reason to add this feature. Ideally you should open a suggestion issue first and have it greenlighted before working on it.
+- *If you are fixing a bug*:
+  - If you are resolving a special issue, please add `(fix #xxxx[,#xxx])` (where `#xxxx` is the GitHub issue ID) to your pull request title, e.g. "`Update documentation (fix #9998)`".
+  - Provide a detailed description of the bug in the pull request. If possible, add a link to a live demo.
+
+### Committing Changes
+
+Possible TODO, see https://github.com/gatsbyjs/gatsby/issues/3920
+
+## Code Guidelines
+
+To ensure consistency throughout the source code, keep these guidelines in mind as you are working:
```

Comment:
Hm. Yeah, I have a list of all sorts of style issues like these. Seems like there are two categories of style: code styling and doc styling.

Reply:
Yep. Without going into things further, we're already using https://github.com/prettier/prettier to consistently format code and most probably don't need this section as we further automate source code formatting—no need to bother the contributors with this.